app/views/publishers/jobseeker_profiles/search/_filters.html.slim

```diff
@@ -0,0 +1,65 @@
+      value_method: :first,
+      selected_method: :last,
+    },
+    {
+      legend: t("publishers.jobseeker_profiles.filters.preferred_key_stages"),
+      key: :key_stages,
+      selected: @form.key_stages,
+      options: @form.key_stage_options,
+      value_method: :first,
+      selected_method: :last,
+    },
+  ]
+
+= filters(submit_button: f.govuk_submit(t("buttons.apply_filters")),
+        filters: { total_count: @jobseeker_profile_search.total_filters },
+        clear_filters_link: { text: t("shared.filter_group.clear_all_filters"), url: publishers_jobseeker_profiles_path(@jobseeker_profile_search.clear_filters_params) },
+        options: { heading_text: "Filter", remove_filter_links: true },
+        html_attributes: { tabindex: "-1", id: "filters-component" }) do |filters_component|
+        - if current_organisation.trust?
+          - filters_component.group key: "location", component: searchable_collection(collection: f.govuk_collection_check_boxes(:locations,
+            f.object.school_options,
+            :first, :last,
+            small: true, legend: { text: t("publishers.jobseeker_profiles.filters.schools") }),
+            collection_count: f.object.school_options.count,
+            options: { scrollable: true })
+        - filters_component.remove_filter_links do |rb|
+          - filter_types.each do |filter_type|
+            - rb.group(**filter_type.merge(remove_filter_link: { url_helper: :publishers_jobseeker_profiles_path, params: @jobseeker_profile_search.filters }))
+        - filters_component.group key: "qualified_teacher_status", component: f.govuk_collection_check_boxes(:qualified_teacher_status, f.object.qts_options, :first, :last, small: true, legend: { text: t("publishers.jobseeker_profiles.filters.qualified_teacher_status") }, hint: nil)
+        - filters_component.group key: "roles", component: f.govuk_collection_check_boxes(:roles, f.object.role_options, :first, :last, small: true, legend: { text: t("publishers.jobseeker_profiles.filters.preferred_roles") }, hint: nil)
+        - filters_component.group key: "working_patterns", component: f.govuk_collection_check_boxes(:working_patterns, f.object.working_pattern_options, :first, :last, small: true, legend: { text: t("publishers.jobseeker_profiles.filters.preferred_working_patterns") }, hint: nil)
+        - filters_component.group key: "education_phases", component: f.govuk_collection_check_boxes(:education_phases, f.object.education_phase_options, :first, :last, small: true, legend: { text: t("publishers.jobseeker_profiles.filters.preferred_education_phases") }, hint: nil)
+        - filters_component.group key: "key_stages", component: f.govuk_collection_check_boxes(:key_stages, f.object.key_stage_options, :first, :last, small: true, legend: { text: t("publishers.jobseeker_profiles.filters.preferred_key_stages") }, hint: nil)
+        = render "shared/subjects_filter", filters_component: filters_component, f: f
```

Comment:
Not like I'm very used to `slim`, but isn't the tabbing spacing a little bit inconsistent across this block? :eyes: